ui/components/ui/form-field/form-field.stories.js

```diff
@@ -57,7 +57,7 @@ export const FormFieldWithTitleDetail = (args) => {
       <button
         style={{
           backgroundColor: clicked
-            ? 'var(--color-secondary-default)'
+            ? 'var(--color-warning-default)'
             : 'var(--color-background-alternative)',
         }}
         onClick={() => setClicked(!clicked)}
```

Comment:
Just a demo component not important
<img width="1440" alt="Screen Shot 2022-05-17 at 2 40 08 PM" src="https://user-images.githubusercontent.com/8112138/168914673-e909daac-9b09-49fb-a09e-79021e99c706.png">